readme.md

```diff
@@ -5,28 +5,29 @@
 
 # Microsoft Azure IoT SDKs
 
-This repository references Microsoft Azure IoT device SDKs and IoT service SDKs. Device SDKs enable you connect client devices to Azure IoT Hub. Service SDKs enable you to manage your IoT Hub service instance.
+This repository serves as a landing page for all Azure IoT SDKs, each being maintained in their own repository, including sample code and documentation.  Visit [this page](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks) to learn more about Azure IoT SDKs.
 
 Visit http://azure.com/iotdev to learn more about developing applications for Azure IoT.
 
 ## Microsoft Azure IoT SDKs
 
-The Microsoft Azure IoT **Device** SDKs contain code that facilitate building devices and applications that connect to and are managed by Azure IoT Hub services.
-The Microsoft Azure IoT **Service** SDKs help you to build applications that interact with your devices and manage device identities in your IoT hub.
+The Microsoft Azure IoT **Device** SDKs:
+- [Azure IoT Device SDK for C](https://github.com/Azure/azure-iot-sdk-c) written in ANSI C (C99) for portability and broad platform compatibility. There are two device client libraries for C, the low-level iothub_client and the serializer.
```

Comment:
This: "There are two device client libraries for C, the low-level iothub_client and the serializer. "

Should be: "There are two device client libraries for C, the low-level iothub_ll_client and the iothub_client (threaded). "

Reply:
I copied it from this doc, should this be updated as well? https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks